airflow/contrib/hooks/sagemaker_hook.py

```diff
@@ -230,61 +231,6 @@ def get_conn(self):
         """
         return self.get_client_type('sagemaker')
 
-    def get_log_conn(self):
-        """
-        Establish an AWS connection for retrieving logs during training
-
-        :rtype: CloudWatchLogs.Client
-        """
-        config = botocore.config.Config(retries={'max_attempts': 15})
-        return self.get_client_type('logs', config=config)
-
-    def log_stream(self, log_group, stream_name, start_time=0, skip=0):
```

Comment:
To maintain back-compat here (so we can include this in 1.10.5) could you include a shim that calls the new hook/method and issues a DeprecationWarning